lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -230,6 +246,44 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Handle $this->_longData after regular query parameters have been bound
+     *
+     * @throws MysqliException
+     */
+    private function _processLongData()
```

Comment:
@carusogabriel do we enforce this in the standard? This is what I'm getting locally for this PR:
```
$ git-phpcs upstream/2.7...c71d64c

FILE: lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 280 | ERROR | [x] There must be exactly 1 whitespace between closing parenthesis and return type
     |       |     colon.
     |       |     (SlevomatCodingStandard.TypeHints.ReturnTypeHintSpacing.IncorrectWhitespaceBeforeColon)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```

Reply:
Yeap, we do. Here’s how our return types should be

    public function foo() : string

Reply:
I've fixed the return type hint (missing space).

Regarding non-public members with a leading underscore, I just followed the practice in the rest of the fields. 🤷‍♂️

Reply:
@carusogabriel I meant the leading underscore in the `_processLongData()` method name. We shouldn't allow it (PSR-2) doesn't.

@mpdude the codebase is full of coding standard violations since fixing some of them (particularly, renaming methods) would lead to BC breaks. We took an incremental approach and only check that the newly added or edited code is compliant with the standard.

Reply:
Ok, but I'm just going to fix the new ones introduced in this PR.

Reply:
@morozov Gotcha. Gonna write some tests for our CS and open a PR there if we don't have this rule :)

Reply:
> Ok, but I'm just going to fix the new ones introduced in this PR.

Sure. This is how our incremental checks work.

> Gotcha. Gonna write some tests for our CS and open a PR there if we don't have this rule :)

@carusogabriel it turns out to be a configuration issue (https://github.com/doctrine/dbal/pull/3287).